examples/online/online-tracker.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { createClient } = require('redis');
+
+/**
+ * Factory function to create an OnlineTracker instance
+ *
+ * @param {Object} redisClient - The Redis client instance.
+ * @returns {Object} An object with methods for user activity tracking.
+ */
+
+function createOnlineTracker(redisClient) {
+  // Method to add a user's activity
+  async function add(user) {
+    const now = Date.now();
+    try {
+      await redisClient.hSet('online_users', user, now.toString());
```

Comment:
I am not sure where this came from (was lifted from the `online` package maybe?), but either way I think this is not the optimal way to achieve this. New versions of redis support expiry on hash keys via `hexpire` which is likely the optimal way to do this. Either way, this is an forever growing hash which is *never* good.

If you think jumping to a new feature is a bad idea (which is a valid opinion I think), then you could add some code to remove the items which are expried in the `last` method.